ParallelBuildsMonitorWindowCommand.cs

```diff
@@ -143,7 +143,6 @@ void BuildEvents_OnBuildProjConfigDone(string Project, string ProjectConfig, str
 
         void BuildEvents_OnBuildBegin(vsBuildScope Scope, vsBuildAction Action)
         {
-            DataModel.CollectPerformanceData(true);   //TODO: I wouldn't collect data here. Problem is that few projects starts in the same moment, so we have overload. I would just get performance statistics in the same time interval.
```

Comment:
Issue 3: For case when 3 builds starts in the same time (because they depend on the same previous project), this will collect data in the same point of time 3 times. Problem is that if performance data is collected several times in the same moment, the first one may vary, but next ones are always 100%. This happen probably because measuring occupy CPU and HDD.
Looks like [Schrödinger's cat](https://en.wikipedia.org/wiki/Schr%C3%B6dinger%27s_cat) issue ;)